packages/formation-react/src/components/modal/Modal.jsx

```diff
@@ -41,14 +41,17 @@ class Modal extends React.Component {
 
   teardownModal() {
     if (this.state.lastFocus) {
-      this.state.lastFocus.focus();
+      setTimeout(() => {
```

Comment:
this seems like a good candidate for a comment explaining why we need this event-loop hack

Reply:
@erikphansen added